server/src/main/java/keywhiz/inject/DataSourceModule.java

```diff
@@ -0,0 +1,20 @@
+package keywhiz.inject;
+
+import com.google.inject.AbstractModule;
+import io.dropwizard.db.ManagedDataSource;
+import keywhiz.service.config.Readonly;
+
+class DataSourceModule extends AbstractModule {
+  private final ManagedDataSource readWrite;
+  private final ManagedDataSource readonly;
+
+  public DataSourceModule(ManagedDataSource readWrite, ManagedDataSource readonly) {
+    this.readWrite = readWrite;
+    this.readonly = readonly;
+  }
+
+  @Override protected void configure() {
+    bind(ManagedDataSource.class).toInstance(readWrite);
+    bind(ManagedDataSource.class).annotatedWith(Readonly.class).toInstance(readonly);
+  }
+}
```

Comment:
This module is used by non-server commands to provide data source bindings.

Technically, with some extra coding on the service's run method (installation of health checks outside of modules) we could use this for both server and non-server commands.